include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -209,6 +210,10 @@ table KUKAiiwaArmConfiguration {
   /// position, cartesian impedence, or joint impedence low level controller adjustments
   controlMode:EControlMode;
 
+  setCartImpedance:CartesianImpedenceControlMode;
```

Comment:
this is for setting the actual parameters for CartesianImpedance. controlMode:EControlMode just sets the mode.